include/camp/resource/omp_target.hpp

```diff
@@ -192,6 +192,11 @@ namespace resources
         }
         return ret;
       }
+      bool compare (Omp* o)
+      {
+        return (dev == o->dev);
+        //TODO: do i need to compare addr???
```

Comment:
Yup, please do.  The value of the address is used by the OpenMP runtime to select a stream or equivalent.